doc/usage/usage.rst

```diff
@@ -2,14 +2,235 @@
 Basic usage
```

Comment:
First of all, thanks for starting to work on documentation. Below you can find a couple of comments and recommendations.

## A hands-on experience

The thing that blocks me currently from starting to play with Permian and the Anaconda WebUI testing is lack of a simple tutorial. I mean a simple copy-paste code that gives me something that works out of the box. A good example of such a code is in the project README (build container, run tests, build docs) - all is easy to execute and runable. I believe this 'how to' style would be great here as well. What about something like this (adding my comments in quote blocks):

> Btw. you may find a bit of inspiration in 'Get Started' section of pytest: https://docs.pytest.org/en/7.2.x/getting-started.html It is very simple and practical, with links at the bottom of the page to get more details.

### How to run a test
> Let's start with something that the user can run locally (if it is possible). I wouldn't complicate things by listing deps if we can use the container.

> I would highly appreciate to provide actual commands, not placeholders like `./pipeline run_event --settings pipeline.ini "event specification"` because sometimes it is quite difficult to figure out how the actual working command looks like.

> It would be also great to demonstrate the best practices - the 1 way to do things right (e.g. use only command X for running tests; use the container instead of installing permian locally).

1. Build container:
    ```
    ./build_container
    ```
1. Do initial setup:
    > e.g. config files, local test library; add more stuff if needed
    ```
    # content of the necessary config file (e.g. pipeline.ini)
    ```
1. Run a concrete test:
    ```
    ./pipeline run_event --settings pipeline.ini '{"type": ... }}'  # something like this, but runable
    ```
    > here may also be a good point to demonstrate dry-run

> Ideally, at this point I feel successful because I was able to run at least 1 actual test (even if it was something more or less artificial just for the sake of this tutorial).

> Bellow come the other sections, now extending the user's experience. Naming just a few you've already provided, feel free to rearrange, rename, add more.

### Run a subset of tests

### Events

### Accessing WebUI

### Extending permian with plugins

Reply:
I've added quick start section, let me know what do you think. The "Next steps" should be updated when we have more detailed documentation.